libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/Matrix/MatrixArea/MatrixArea.styles.ts

```diff
@@ -21,6 +22,11 @@ export const matrixAreaStyles: () => IProcessedStyleSet<IMatrixAreaStyles> =
       emptyLabelPadding: {
         paddingTop: "60px"
       },
+      layerHost: {
+        height: "500px",
```

Comment:
Since height is fixed, does this work in zoom-in conditions?

Reply:
Great point, but we specify the height in pixels in many other places as well, so if it's a problem we need a more general fix probably.  However, so far I haven't heard that this is a problem in either the widget or full-screen view.  Perhaps we may need to change this eventually.  I'm not quite sure what the ideal fix is - do we want to provide percentages everywhere instead?